### PR TITLE
📖 Add teardown instructions to examples.md

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR extends examples.md with instructions on how to teardown each scenario without un-deploying KubeStellar.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by ezrasilvera** [comment]
@MikeSpreitzer Your assumption is that teardown should bring the user to the point before he started the specific example or before he deployed all the KS env -  KS+KF+...

**Comment by MikeSpreitzer** [comment]
The teardown for each scenario is intended to undo just that scenario.

**Comment by MikeSpreitzer** [comment]
I have successfully tested this.

File: docs/content/direct/examples.md [hunk 1]
```diff
@@ -594,6 +658,15 @@ kubectl --context cluster1 get deployments -n nginx-res
...
 ```
 
+## Teardown Scenario 5
+
+```shell
+kubectl --context wds1 delete ns nginx-res
+kubectl --context wds1 delete bindingpolicies nginx-res-bpolicy
+```
+
+The continue with [teardown of scenario 1](#teardown-scenario-1).
```
Comment (ezrasilvera) [inline docs/content/direct/examples.md:668]: Not clear . Is it "Then" ?

**Comment by ezrasilvera** [comment]
@MikeSpreitzer Maybe add "optional"  on each tear down step ?

File: docs/content/direct/examples.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/direct/examples.md]: yes, typo. Fixed. Thanks.

**Comment by MikeSpreitzer** [comment]
The force-push to 9fde3019f fixes the typo noted above and marks the teardown sections as "[Optional]".

File: docs/content/direct/examples.md [hunk 2]
```diff
@@ -357,6 +366,48 @@ kubectl --context cluster1 get appwrappers
...
+```
+
+Wait until the following commands show no appwrappers in cluster1 and cluster2.
+
+```shell
+kubectl --context cluster1 get appwrappers -A
+kubectl --context cluster2 get appwrappers -A
+```
+
+The continue.
```
Comment (effi-ofer) [inline docs/content/direct/examples.md:383]: "The continue" ?

Comment (MikeSpreitzer) [reply docs/content/direct/examples.md:383]: Oops, typo. s/the/then/

**Pull request merged**